src/OpenTelemetry/Internal/SelfDiagnosticsEventListener.cs

```diff
@@ -55,11 +55,7 @@ public SelfDiagnosticsEventListener(EventLevel logLevel, SelfDiagnosticsConfigRe
 
             foreach (var eventSource in eventSources)
             {
-#if NET452
-                this.EnableEvents(eventSource, this.logLevel, (EventKeywords)(-1));
```

Comment:
Finally 👍 

IIRC there are many places where we have `(EventKeywords)(-1)`, probably not fully covered by this PR. Can be a separate PR though.